Assets/Scripts/Checkpoint.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Animations;
+
+public class Checkpoint : MonoBehaviour {
+    [SerializeField]
+    public bool isActivated = false;
+   
+    public Animator anim;
+
+    //creates a variable that refers to the PlayerController
+    PlayerController player;
+
+    private void Start()
+    {
+        anim = this.GetComponent<Animator>();       
```

Comment:
the `this` here is not necessary. Normally studios will either always use it (for clarity) or only use it when it's necessary (for brevity). I guess I normally only use it when necessary. Pick a standard and stick to it. I guess since we're in my class I'd say don't use it unless it's necessary.

Reply:
I'm usually the same, my programming teacher always uses it and I've been trying to keep it from rubbing off on me but I guess it slipped that time

Reply:
I'm usually the same, my programming teacher always uses it and I've been trying to keep it from rubbing off on me but I guess it slipped that time